src/Sylius/Bundle/PaymentBundle/Resources/config/doctrine/model/PaymentRequest.orm.xml

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<!--
+
+ This file is part of the Sylius package.
+
+ (c) Sylius Sp. z o.o.
+
+ For the full copyright and license information, please view the LICENSE
+ file that was distributed with this source code.
+
+-->
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:gedmo="http://gediminasm.org/schemas/orm/doctrine-extensions-mapping">
+
+    <mapped-superclass name="Sylius\Component\Payment\Model\PaymentRequest" table="sylius_payment_request">
+        <id name="hash" column="hash" type="uuid">
+            <generator strategy="CUSTOM" />
+            <custom-id-generator class="Ramsey\Uuid\Doctrine\UuidGenerator"/>
+        </id>
+
+        <many-to-one field="method" target-entity="Sylius\Component\Payment\Model\PaymentMethodInterface">
+            <join-column name="method_id" referenced-column-name="id" nullable="true" />
+        </many-to-one>
+
+        <many-to-one field="payment" target-entity="Sylius\Component\Payment\Model\PaymentInterface">
+            <join-column name="payment_id" referenced-column-name="id" nullable="true" />
+        </many-to-one>
```

Comment:
Yes, but I did not try to set `on-delete="CASCADE"`, I remember that it will delete the payment request only when a payment is delete, right? not the opposite?

Reply:
I also forgot which is which and had to check 💃 
Yes, `on-delete=CASCADE` will delete requests when the payment gets removed,
`cascade={"remove"}` does the opposite.